File: practice/data_structures/trees/inorder_traversal/inorder_traversal.cpp
/* you only have to complete the function given below.  
Node is defined as  

struct node
{
    int data;
    node* left;
    node* right;
};

*/

void inOrder(node * i_root)
{
    if (i_root)
    {
        inOrder(i_root->left);
        cout << i_root->data << " ";
        inOrder(i_root->right);
    }
}

File: practice/data_structures/trees/inorder_traversal/inorder_traversal.py
"""
Node is defined as
self.left (the left child of the node)
self.right (the right child of the node)
self.data (the value of the node)
"""

def inOrder(i_root):
    if i_root:
        inOrder(i_root.left)
        print i_root.data,
        inOrder(i_root.right)

